mongodbatlas/provider_test.go

```diff
@@ -159,6 +159,30 @@ func testCheckAwsEnv(tb testing.TB) {
 	}
 }
 
+func testCheckSTSAssumeRole(tb testing.TB) {
```

Comment:
do we need all these variables in the test?

Reply:
yes because that is how we generate credentials via STS Assume Role.